src/main/java/domain/December.java

```diff
@@ -1,60 +1,24 @@
 package domain;
 
-import java.time.LocalDateTime;
-import java.util.Arrays;
 import java.util.List;
 
 public enum December {
+    HOLIDAY(List.of(25)),
+    WEEKDAY(List.of(2, 3, 4, 5, 6, 9, 10, 11, 12, 13, 16, 17, 18, 19, 20, 23, 24, 26, 27, 30, 31)),
+    WEEKEND(List.of(1, 7, 8, 14, 15, 21, 22, 28, 29));
 
-    MONDAY(List.of(2, 9, 16, 23, 30), "월요일"),
-    TUESDAY(List.of(3, 10, 17, 24, 31), "화요일"),
-    WEDNESDAY(List.of(4, 11, 18), "수요일"),
-    THURSDAY(List.of(5, 12, 19, 26), "목요일"),
-    FRIDAY(List.of(6, 13, 20, 27), "금요일"),
-    SATURDAY(List.of(7, 14, 21, 28), "토요일"),
-    SUNDAY(List.of(8, 15, 22, 29), "일요일"),
-    HOLIDAY(List.of(25), "휴일");
-
-    public static final int DEFAULT_YEAR = 2024;
-    public static final int DEFAULT_MONTH = 12;
-    public static final int DECEMBER_WEEKDAY_COUNTS=21;
-    
-    private final List<Integer> dates;
-    private final String dayOfWeek;
-
-    December(List<Integer> dates, String dayOfWeek) {
-        this.dates = dates;
-        this.dayOfWeek = dayOfWeek;
-    }
-
-    public static December findDayOfWeek(int dayOfMonth) {
-        return Arrays.stream(December.values()).filter(day -> day.dates.contains(dayOfMonth)).findFirst().orElseThrow(() -> new IllegalArgumentException("[]"));
+    private final List<Integer> days;
 
+    December(List<Integer> days) {
+        this.days = days;
     }
 
-    public static List<Integer> getWeekDays() {
-        return Arrays.stream(December.values())
-                .filter(a -> !List.of(SUNDAY, SATURDAY, HOLIDAY).contains(a))
-                .flatMap(a -> a.dates.stream()).sorted().toList();
+    public static boolean checkHoliday(int dayOfMonth) {
```

Comment:
```suggestion
    public static boolean isHoliday(int dayOfMonth) {
```
boolean 값을 반환하는 메서드명은 is, has 로 나타내주면 좋습니다. 😄 [참고](https://tecoble.techcourse.co.kr/post/2020-04-26-Method-Naming/)
코드 전체적으로 메서드명을 확인해주시면 좋을 것 같아요 😄

Reply:
참고 자료에 대해서 추가로 학습해보았습니다!

메서드가 어떤 이름을 가져야할지 잘 나와 있는 것 같습니다!

메서드명을 지을때 고민이 많았는데 이번 기회에 잘 공부할 수 있었던 것 같습니다!

감사합니다!👍